live-examples/js-examples/promise/promise-constructor.html

```diff
@@ -1,9 +1,23 @@
 <pre>
 <code id="static-js">var promise1 = new Promise(function(resolve, reject) {
-  setTimeout(resolve, 100, 'foo');
+  setTimeout(function() { 
+    resolve('foo');
+  }, 100);
 });
 
+
+// How to show the value of promise1: 
+
+promise1.then(value => console.log(value));
```

Comment:
I know this is the `constructor` example- I feel this bit is needed because people see `constructor` in their search, then think assume they will find how to make (and perhaps read) promises there." Well... Not exactly on this page. 😿 

While there are good supporting links and contextual info, it can easily be missed, especially the first time you land on the example/page.

The `.then()` line is a far more common use case, and I hope will benefit many users. 🤞 

> Note: I wouldn't add anything extraneous like `.catch()`

Reply:
Yeah, I can see the value of showing how to use the promise in this example.

(really I suspect about 95% of the people who land on this page should be reading https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises instead)